packages/web-components/src/search-widget/server/api/get-properties.ts

```diff
@@ -4,19 +4,34 @@ import { PagedResultPropertyModel_ } from '@reapit/foundations-ts-definitions'
 import { errorHandler } from '../../../common/utils/error-handler'
 import { getServerHeaders } from '../../../common/utils/get-server-headers'
 import { PACKAGE_SUFFIXES } from '../../../common/utils/constants'
+import { mapMinimalProperties } from '../utils/map-minimal-properties'
+import { PickedPagedResultPropertyModel_ } from '../../types'
 
 export const getProperties = async (req: Request, res: Response) => {
   try {
     const headers = await getServerHeaders(req, PACKAGE_SUFFIXES.SEARCH_WIDGET)
     const url = new URL(`${process.env.PLATFORM_API_BASE_URL}${req.url}`)
-    const refreshResponse = await fetcher<PagedResultPropertyModel_, undefined>({
+    const fullPagedResult = await fetcher<PagedResultPropertyModel_, undefined>({
       url: String(url),
       headers,
     })
 
-    if (refreshResponse) {
+    if (fullPagedResult) {
+      const includedProps = [
```

Comment:
Can we move to a constant file so the same list is included in the unit tests - there is a good chance this list will be added to so would be nice if the test can be accurate for free in this case.